### PR TITLE
Move settings

### LINKED ISSUE #211: Move check in/out settings to Bookings settings
The check in/out times are set under WooCommerce > Settings > Products > Accommodations, this should probably be moved to Bookings > Settings > Accommodations to clear up confusion. 

-------------------

- [ ] Issue assigned to next milestone.
- [ ] Issue assigned a priority (will be assessed by maintainers).

### PR DESCRIPTION
**Pull request by bor0**
Fixes #211.

### CODE REVIEW

File: includes/admin/class-wc-accommodation-booking-admin-product-settings.php [hunk 1]
```diff
@@ -5,101 +5,229 @@
...
 	public function __construct() {
+		$this->maybe_migrate();
+		$this->plugin_id = "woocommerce_accommodation_bookings_";
+		$this->id = "times";
+
+		// Initialize settings and form data.
+		$this->init_times_settings();
+
+		add_action( 'admin_init', array( $this, 'maybe_save_settings' ) );
+		add_filter( 'woocommerce_bookings_settings_page', array( $this, 'add_accommodation_settings' ) );
```
Comment (mikkamp) [inline includes/admin/class-wc-accommodation-booking-admin-product-settings.php:48]: I'm not finding the matching PR / commit in Bookings where this filter was added. So it never shows the tab.

File: includes/admin/class-wc-accommodation-booking-admin-product-settings.php [hunk 2]
```diff
@@ -5,101 +5,229 @@
...
+		}
+
+		$check_out = get_option( 'woocommerce_accommodation_bookings_check_out' );
+
+		if ( $check_out ) {
+			delete_option( 'woocommerce_accommodation_bookings_check_out' );
+		}
+
+		if ( $check_in || $check_out ) {
+			update_option( $this->plugin_id . $this->id . '_settings', array(
```
Comment (mikkamp) [inline includes/admin/class-wc-accommodation-booking-admin-product-settings.php:68]: `plugin_id` and `id` get set after the call to `maybe_migrate`. So my settings end up being saved with the key `woocommerce__settings`

File: includes/admin/class-wc-accommodation-booking-admin-product-settings.php (hunk 1 above)
Comment (bor0) [reply includes/admin/class-wc-accommodation-booking-admin-product-settings.php:48]: Ahh, sorry, I have it locally but forgot to commit. Pushed to `trunk` in Bookings.

File: includes/admin/class-wc-accommodation-booking-admin-product-settings.php (hunk 2 above)
Comment (bor0) [reply includes/admin/class-wc-accommodation-booking-admin-product-settings.php:68]: Nice catch. I had tested it prior to moving the call. Did another test now and works fine.

**Review by mikkamp — APPROVED** [review]
Looks good, it's all working now.

**Pull request merged**